packages/live-share-inking/src/core/Colors.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the Microsoft Live Share SDK License.
+ */
+
+/**
+ * Defines an RGB color
+ */
+ export interface IColor {
+    readonly r: number; // 0 - 255
+    readonly g: number; // 0 - 255
+    readonly b: number; // 0 - 255
+}
+
+/**
+ * Converts an IColor object into its CSS representation.
+ * @param color The color to convert.
+ * @returns A string representing the CSS representation of the color.
+ */
+ export function toCssColor(color: IColor, alpha: number = 1): string {
+    return `rgba(${color.r},${color.g},${color.b},${alpha})`;
+}
+
+/**
+ * Converts a CSS color expressed in the #RRGGBB[AA] format into an IColor object
+ * @param color The CSS color to convert.
+ * @returns The converted color as an IColor object, or a default color if the provided
+ * color string isn't valid.
+ */
+export function fromCssColor(color: string): IColor {
+    if (color) {
+        const regEx = /#([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})/gi;
+        const matches = regEx.exec(color);
+
+        if (matches) {
+            const r = parseInt(matches[1], 16);
+            const g = parseInt(matches[2], 16);
+            const b = parseInt(matches[3], 16);
+
+            return { r, g, b };
+        }
+    }
+
+    return penColors.black;
+}
+
+/**
+ * Lightens the given color by a certain intensity.
+ * @param color The color to lighten.
+ * @param intensity The intensity of the lightening. Must be between 0 and 100.
+ * @returns The lightened color.
+ */
+export function lightenColor(color: IColor, intensity: number): IColor {
+    if (intensity < 0 || intensity > 100) {
+        return color;
+    }
+
+    const brightenChannel = (channel: number) => {
+        const delta = 255 - channel;
+
+        return channel + delta / 100 * intensity;
+    }
+
+    return {
+        r: brightenChannel(color.r),
+        g: brightenChannel(color.g),
+        b: brightenChannel(color.b)
+    }
+}
+
+/**
+ * Darkens the given color by a certain intensity.
+ * @param color The color to darken.
+ * @param intensity The intensity of the darkening. Must be between 0 and 100.
+ * @returns The darkened color.
+ */
+ export function darkenColor(color: IColor, intensity: number): IColor {
+    if (intensity < 0 || intensity > 100) {
+        return color;
+    }
+
+    const darkenChannel = (channel: number) => {
+        const delta = 255 - channel;
+
+        return channel - channel / 100 * intensity;
+    }
+
+    return {
+        r: darkenChannel(color.r),
+        g: darkenChannel(color.g),
+        b: darkenChannel(color.b)
+    }
+}
+
+/**
+ * Pre-defined basic colors
+ */
+export const basicColors = {
```

Comment:
>basicColors [](http://example.com/codeflow?start=13&length=11)

This is inconsistent with the casing of other constants & enums in the SDK. All consts are treated like enums and strat with capital letters. It's fine to camel case the values themselves.

Reply:
Fixed in https://github.com/microsoft/live-share-sdk/commit/18b13a631f0c2833a77b7d5181cb89b36206c512